### PR TITLE
Added package.json for management, config.json for config, and took out the router part of the constructor

### PR DESCRIPTION
**Pull request by gilrosenthal**

### CODE REVIEW

**Comment by gilrosenthal** [comment]
The router was not doing anything and was seeming to cause errors, so i just took it out

File: config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "slacktoken":"",
```
Comment (ryanw-se) [inline config.json]: Just for readability, this should be changed to ```slack_token```.

File: config.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    "name":"kira",
+    "witaitoken":""
```
Comment (ryanw-se) [inline config.json]: Same here, this should be changed to ```witai_token```, just to make things more readable.

File: README.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ Change to the Kira directory, install the dependencies:
 
 You need to create a Slack bot, or input the API key of the account you’d like the bot to run on. You must have integration creation permissions (ask your Slack team admin if you don’t know what that is), and go to [the new bot integration page](https://my.slack.com/services/new/bot). Name your bot, press “Add Bot Integration”, and you’ll be taken to a page with an API token. **Copy it**, you’ll need it. 
 
-You now need to edit the *api.txt* file with the key of where you’d like the bot to run. Paste the token you just copied, and ensure the file is only one line long.
+You now need to edit the *config.json* file with the key of where you’d like the bot to run. Paste the token you just copied into the `slacktoken` property, and make sure it is a valid key.
```
Comment (ryanw-se) [inline README.md]: This needs to be updated to reflect ```slack_token```, once the other changes are implemented.

File: README.md [hunk 4]
```diff
@@ -21,9 +21,9 @@ Change to the Kira directory, install the dependencies:
...
 
-Then, you need to go to Wit.ai and log in with your GitHub account. Once you’ve logged in, you need to fork [the wit.ai instance](https://wit.ai/tnws/kira). Then, go to your application settings and copy the API token. With that token, go back to the Kira folder. Go to *details/kira.coffee*. Fill in the `token` variable, by adding in your token within the quotes. 
+Then, you need to go to Wit.ai and log in with your GitHub account. Once you’ve logged in, you need to fork [the wit.ai instance](https://wit.ai/tnws/kira). Then, go to your application settings and copy the API token. With that token, go back to *config.json*. Fill in the `witaitoken` variable, by adding in your token within the quotes.
```
Comment (ryanw-se) [inline README.md]: Same here, this will also need updated to reflect ```witai_token```, once the other changes are implemented.

**Comment by ryanw-se** [comment]
Added some feedback, I also think it would be a good idea to add a little segment in the README detailing each of the configuration options within the ```config.json``` file, as currently they're strung about in a not-so-organized manner.

**Comment by gilrosenthal** [comment]
Yeah sounds good, I'll revise in a bit

**Comment by tnws** [comment]
@ryanw-se It was apart of some old code that was never removed, yeah.

**Pull request closed without merging**